build_library/sysext_mangle_flatcar-zfs

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -euo pipefail
+rootfs="${1}"
+
+pushd "${rootfs}"
+
+rm -rf ./usr/{lib/debug/,lib64/cmake/,include/}
+rm -rf ./usr/lib/dracut/
+rm -rf ./usr/share/initramfs-tools
+rm -rf ./usr/src
+
+mkdir -p ./usr/share/flatcar/etc
+rm -rf ./etc/{csh.env,environment.d/,profile.env}
+cp -a ./etc/. ./usr/share/flatcar/etc/
```

Comment:
I fear that this may be confusing - how about a different path?
The use of `/usr/lib/systemd/system/systemd-udevd.service.d/10-zfs.conf` is ok because this would ideally work once we set up the overlay early enough but putting files under `/usr/share/flatcar/etc/` won't make them show up.
```suggestion
cp -a ./etc/. ./usr/share/zfs/etc/
```

Reply:
make sense